lighthub/main.cpp

```diff
@@ -536,191 +557,198 @@ void Changed (int i, DeviceAddress addr, int val)
+    strcpy_P(addrstr, outprefix);
+    strncat(addrstr, addrbuf, sizeof(addrstr));
     //snprintf(addrstr,sizeof(addrstr),"%s%s",F(outprefix),addrbuf);
-    sprintf(valstr,"%d",val);
-    client.publish(addrstr, valstr);
+    sprintf(valstr, "%d", val);
+    mqttClient.publish(addrstr, valstr);
 
-    if (owItem)
-    {
-        thermoSetCurTemp(owItem,val);  ///TODO: Refactore using Items interface
+    if (owItem) {
+        thermoSetCurTemp(owItem, val);  ///TODO: Refactore using Items interface
     }
 }
+
 #endif
 
-void modbusIdle(void) ;
+void modbusIdle(void);
 
 void _handleHelp(int arg_cnt, char **args)
 //(char* tokens)
 {
-    Serial.println(F("Use the commands: 'help' - this text\n'save' - save config in NVRAM\n'get' - get config from pre-configured URL\n'load' - load config from NVRAM\n'kill' - test watchdog"));
+    Serial.println(F("Use the commands: 'help' - this text\n"
+                             "'set de:ad:be:ef:fe:00' set and store MAC-address in EEPROM\n"
+                             "'save' - save config in NVRAM\n"
+                             "'get' - get config from pre-configured URL\n"
+                             "'load' - load config from NVRAM\n"
+                             "'kill' - test watchdog"));
 }
 
-void _kill(int arg_cnt, char **args)
-{
-    for (short i=17;i>0;i--) {delay(1000);Serial.println(i);};
+void _kill(int arg_cnt, char **args) {
+    for (short i = 17; i > 0; i--) {
+        delay(1000);
+        Serial.println(i);
+    };
 }
 
 #define EEPROM_offset 32+6
 
-void parseConfig()
-{    int mc,incnt;
+void parseConfigFromParsedJSON() {
+    int arrayItemValue, itemsCount;
     //DMX out is configured
-    aJsonObject *dmxoutArr = aJson.getObjectItem(root, "dmx");
+    aJsonObject *dmxoutArr = aJson.getObjectItem(aJsonObjectRoot, "dmx");
 #ifdef _dmxout
-    if (dmxoutArr && aJson.getArraySize(dmxoutArr)==2)
-    {
-        DMXoutSetup(mc=aJson.getArrayItem(dmxoutArr,1)->valueint,aJson.getArrayItem(dmxoutArr,0)->valueint);
+    if (dmxoutArr && aJson.getArraySize(dmxoutArr) == 2) {
+        DMXoutSetup(arrayItemValue = aJson.getArrayItem(dmxoutArr, 1)->valueint,
+                    aJson.getArrayItem(dmxoutArr, 0)->valueint);
         Serial.print(F("DMX out started. Channels: "));
-        Serial.println(mc);
+        Serial.println(arrayItemValue);
     }
 #endif
     //DMX in is configured
 #ifdef _dmxin
-    dmxArr=    aJson.getObjectItem(root, "dmxin");
-    if (dmxArr && (incnt=aJson.getArraySize(dmxArr)))
-    {
-        DMXinSetup(incnt*4);
+    dmxArr = aJson.getObjectItem(aJsonObjectRoot, "dmxin");
+    if (dmxArr && (itemsCount = aJson.getArraySize(dmxArr))) {
+        DMXinSetup(itemsCount * 4);
         Serial.print(F("DMX in started. Channels:"));
-        Serial.println(incnt*4);
+        Serial.println(itemsCount * 4);
     }
 #endif
 
-    items =    aJson.getObjectItem(root,"items");
+    items = aJson.getObjectItem(aJsonObjectRoot, "items");
```

Comment:
pls revert "root" back
I'm really used to do it

Reply:
May be find some better name? Root is like a,b,c name. Doesn't say anything special.